enumerable.rb

```diff
@@ -0,0 +1,120 @@
+# rubocop :disable Style/CaseEquality
+
+# my_each
+module Enumerable
+  def my_each
+    return to_enumerable(:my_each) unless my_each_given?
+
+    each do |x|
+      yield x
+    end
+    my_each
+  end
+  # end of my_each
+
+  # #my_each_with_index
+  def my_each_with_index
+    return to_enumerable(:my_each_with_index) unless my_each_with_index_given?
```

Comment:
- [ ] on line #17 you use my_each_with_index_given? method but this method is not defined in your Enumerable module, let's define it or use a different way to make this validation work